### PR TITLE
adding subscription_id in invoice table for use in the main stripe pa…

### PR DESCRIPTION
**Pull request by ashishk-de**
…ckage

Pull Request
**Are you a current Fivetran customer?** 
yes. Abine Inc

**What change(s) does this PR introduce?** 
Adds subscription_id to invoice table. 

**Did you update the CHANGELOG?** 

- [X] Yes

**Does this PR introduce a breaking change?**
- [ ] Yes (please provide breaking change details below.)
- [ ] No  (please provide an explanation as to how the change is non-breaking below.)
- [X] Unsure  (please provide an explanation as to how the change is non-breaking below.)
My stripe instance uses subscription. I am unsure if the Invoice object from stripe will always have the subscription_id column available based on your usage of subscriptions feature in stripe. 

**Did you update the dbt_project.yml files with the version upgrade (please leverage standard semantic versioning)? (In both your main project and integration_tests)** 

- [x] Yes

**Is this PR in response to a previously created Bug or Feature Request**

- [ ] Yes, Issue/Feature #44 
- [X] No 

**How did you test the PR changes?** 

 

- [ ] CircleCi  
- [X] Local (please provide additional testing details below)
image

**Select which warehouse(s) were used to test the PR**
 

- [ ] BigQuery
- [ ] Redshift
- [X] Snowflake
- [ ] Postgres
- [ ] Databricks
- [ ] Other (provide details below)

**Provide an emoji that best describes your current mood**
 
:dancer:

**Feedback**

We are so excited you decided to contribute to the Fivetran community dbt package! We continue to work to improve the packages and would greatly appreciate your [feedback](https://www.surveymonkey.com/r/DQ7K7WW) on our existing dbt packages or what you'd like to see next.

### CODE REVIEW

File: dbt_project.yml
Comment (fivetran-joemarkiewicz) [inline dbt_project.yml:42]: We should probably leave this as `false` by default still.

**Review by fivetran-joemarkiewicz — CHANGES_REQUESTED** [review]
@ashishk-de thanks so much for opening this PR 🙌 

I apologize for the delayed response in giving this a review. I just took a look and this enhancement seems like a more than reasonable addition to the package. Before we progress with merging this to `main` and including in the next release, I did have a few requested updates:
- [ ] With the new field being added to the staging model, we should also update the `src_stripe.yml` and `stg_stripe.yml` files to include the new field within the dbt documentation.
- [ ] Make the same version increase adjustment that you made in the `dbt_project.yml` file within the `integration_test/dbt_project.yml` file.

Once those small changes are applied, we should be good to go! Let me know if you have any questions 😄

**Comment by fivetran-jamie** [comment]
closing as this was merged with another PR

**Pull request closed without merging**
